Shuttle-front/src/app/app.module.ts

```diff
@@ -6,39 +6,50 @@ import { AppRoutingModule } from 'src/infrastructure/app-routing.module';
 import { AppComponent } from './app.component';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
+import {MatToolbarModule} from '@angular/material/toolbar';
+import {MatButtonModule} from '@angular/material/button';
+import {MatInputModule} from '@angular/material/input';
 import { RegisterComponent } from './register/register.component'
 
-import { LoginComponent } from './login/component/login/login.component';
-import { NavbarComponent } from './navbar/navbar.component';
+import { LoginComponent } from './auth/login/component/login/login.component';
+import { ReactiveFormsModule } from '@angular/forms';
+import { UnregisteredNavbarComponent } from './navbar-module/unregistered-navbar/unregistered-navbar.component';
 import { MaterialModule } from 'src/infrastructure/material.module';
 import { AdminModule } from './admin/admin.module';
-import { ForgotPasswordComponent } from './login/component/forgot-password/forgot-password.component';
-import { ResetPasswordComponent } from './login/component/reset-password/reset-password.component';
+import { ForgotPasswordComponent } from './auth/login/component/forgot-password/forgot-password.component';
+import { ResetPasswordComponent } from './auth/login/component/reset-password/reset-password.component';
 import { SharedModule } from './shared/shared.module';
 import { DriverProfileComponent } from './driver-profile/driver-profile.component';
 import { DriverHomeComponent } from './driver/driver-home/driver-home.component';
 import { RejectRideDialogComponent } from './driver/reject-ride-dialog/reject-ride-dialog.component';
+import {NavbarModuleModule} from "./navbar-module/navbar-module.module";
 
 @NgModule({
     declarations: [
         AppComponent,
         RegisterComponent,
         LoginComponent,
-        NavbarComponent,
         ForgotPasswordComponent,
         ResetPasswordComponent,
         DriverProfileComponent,
         DriverHomeComponent,
         RejectRideDialogComponent,
+        LoginComponent
     ],
     imports: [
+        BrowserModule,
         HttpClientModule,
         AppRoutingModule,
         BrowserModule,
         BrowserAnimationsModule,
         AdminModule,
         SharedModule,
-        MaterialModule
+        MaterialModule,
+        MatToolbarModule,
+        MatButtonModule,
+        ReactiveFormsModule,
+        MatInputModule,
```

Comment:
All modules related to Angular Material should go in `MaterialModule` so as not to clutter `app.module.ts`.